cmd/show_test.go

```diff
@@ -226,5 +226,61 @@ func TestShowExec(t *testing.T) {
 	if got != want {
 		t.Fatalf("got: %q, want: %q", got, want)
 	}
+}
+
+func TestShowOverlay(t *testing.T) {
```

Comment:
Could this be converted into a table-test to reduce the boilerplate code around the multiple instances of `got := ... want := ...`

Not a huge deal since it is still clear what is going on and it looks like there is some revisiting anyway with the `TODO(mkm) ...` comment :blush:

Reply:
I'm a huge fan of table driven tests but in this case I think it doesn't pay the cost of its boilerplate.